__tests__/app.test.js

```diff
@@ -118,6 +119,32 @@ describe('GET Requests', () => {
                         article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
                         comment_count: 2
                     })
+                });
+         });
+    });
+    describe('GET /api/articles/:article_id', () => {
+        test('should return a sigle article with the article_id coresponding to the request.', () => {
+            return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.article[0].article_id).toBe(1);
+                });
```

Comment:
Please add a test to check all of the properties in the article as well as the article_id.